### PR TITLE
Fixing error log formatting

### PR DESCRIPTION
**Pull request by jpodivin**

### CODE REVIEW

File: modules/common/job/job.go [hunk 1]
```diff
@@ -291,7 +291,7 @@ func GetJobWithName(
 	job := &batchv1.Job{}
 	err := h.GetClient().Get(ctx, types.NamespacedName{Name: name, Namespace: namespace}, job)
 	if err != nil {
-		h.GetLogger().Info("GetJobWithName %s err: %w", name, err)
+		h.GetLogger().Info(fmt.Sprintf("GetJobWithName %s err: %s", name, err))
 		return job, err
```
Comment (stuggi) [inline modules/common/job/job.go:295]: I think its better to use `return job, fmt.Errorf("GetJobWithName %s err: %w", name, err)`

Comment (jpodivin) [reply modules/common/job/job.go:295]: I suppose.

Comment (jpodivin) [reply modules/common/job/job.go]: Done.

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**